phoenix-core/src/it/java/org/apache/phoenix/end2end/JsonArrayLengthFunctionIT.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * 
+ */
+package org.apache.phoenix.end2end;
+
+import static org.apache.phoenix.util.TestUtil.TEST_PROPERTIES;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Properties;
+import org.apache.phoenix.expression.function.JsonArrayLengthFunction;
+import org.apache.phoenix.util.PropertiesUtil;
+import org.junit.Test;
+
+/**
+ * End to end test for {@link JsonArrayLengthFunction}.
+ *
+ */
+public class JsonArrayLengthFunctionIT extends BaseHBaseManagedTimeIT {
+
+	@Test
+	public void testJsonArrayLengthWithWhereClause() throws Exception {
```

Comment:
Can you test with more complex json (like nested json and the different data types)